examples/riot-kitchen-sink/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "riot-example",
+  "version": "4.0.0-alpha.16",
+  "private": true,
+  "scripts": {
+    "build": "cross-env NODE_ENV=production webpack --progress --hide-modules",
+    "build-storybook": "build-storybook -s public",
+    "dev": "cross-env NODE_ENV=development webpack-dev-server --open --hot",
+    "storybook": "start-storybook -p 9009 -s public"
+  },
+  "dependencies": {
+    "riot-hot-reload": "^1.0.0",
+    "riot-compiler": "^3.5.1",
+    "riot": "^3.11.1"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.0.0-rc.2",
+    "@storybook/addon-actions": "4.0.0-alpha.16",
+    "@storybook/addon-backgrounds": "4.0.0-alpha.16",
+    "@storybook/addon-centered": "4.0.0-alpha.16",
+    "@storybook/addon-knobs": "4.0.0-alpha.16",
+    "@storybook/addon-links": "4.0.0-alpha.16",
+    "@storybook/addon-notes": "4.0.0-alpha.16",
+    "@storybook/addon-options": "4.0.0-alpha.16",
+    "@storybook/addon-storyshots": "4.0.0-alpha.16",
+    "@storybook/addon-storysource": "4.0.0-alpha.16",
+    "@storybook/addon-viewport": "4.0.0-alpha.16",
+    "@storybook/addons": "4.0.0-alpha.16",
+    "@storybook/riot": "../../app/riot",
```

Comment:
should be a version as well, as the above packages, yarn workspaces makes this work.